tasks/reduce-snapshot/README.md

```diff
@@ -0,0 +1,13 @@
+# reduce-snapshot
+
+Tekton task to reduce a snapshot to a single component based on the component that the snapshot was built for.
+
+## Parameters
+
+| Name                                | Description                                              | Optional | Default value |
+|-------------------------------------|----------------------------------------------------------|----------|---------------|
+| SNAPSHOT                            | String representation of Snapshot                        | No       | -             |
```

Comment:
So this is just the spec? Can you say that in the description then? That is what I am gathering from one of the threads at least

Reply:
updated description